examples/fleet.yaml

```diff
@@ -19,17 +19,7 @@ spec:
           gitRef:
             repository: defaultRepo
             targetRevision: main
-            path: /production
+            path: /basic-nginx-demo/configuration
         - name: motd-update
           configType: InlineConfigProviderSpec
-          inline:
-            ignition:
-              version: 3.4.0
-            storage:
-              files:
-                - contents:
-                    source: >-
-                      data:,This%20system%20is%20managed%20by%20flightctl.%0A
-                  mode: 422
-                  overwrite: true
-                  path: "/etc/motd"
+          inline: "{\"ignition\": {\"version\": \"3.4.0\"}, \"storage\": {\"files\": [{\"path\": \"/etc/motd\", \"mode\": 422, \"overwrite\": true, \"user\": { \"name\": \"root\"},\"contents\": {\"source\": \"data:,This%20system%20is%20managed%20by%20flightctl.%0A\"}}]}}"
```

Comment:
Why do we change the yaml into json?

json has some disadvantages, for example the mode cannot be written in octal 0422, because that's not supported by json, it's also harder to manipulate from a yaml file.

Reply:
The code generation didn't give yaml, it gave maps to interfaces and arrays of interfaces, and it was just a mess to translate that into an actual yaml string.  I'll see how to manage it.